src/features/board/Board.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { board, attack } from './boardSlice';
+import {selectCard, unselectCard, selectSelectedCard} from '../card/selectedCardSlice';
 import { useAppSelector, useAppDispatch } from '../../app/hooks';
 import { CardInterface, Card } from '../card/Card';
+import {useSelector} from "react-redux";
 
 export function Board () {
     const playerCards: CardInterface[] = useAppSelector(board).playerSide.cards;
     const opponentCards: CardInterface[] = useAppSelector(board).opponentSide.cards;
     const dispatch = useAppDispatch();
+    const selectedCardPosition = useSelector(selectSelectedCard);
+
+    useEffect(() => {
+        window.addEventListener('mouseup', handleDeselectCard);
+        return () => {
+            window.removeEventListener('mouseup', handleDeselectCard);
+        };
+    }, [selectedCardPosition]);
+
+    const handleSelectCard = (cardPosition:number) => {
+        if (null === selectedCardPosition) {
```

Comment:
- For me, it is really difficult to read Yoda conditions. I'll accept it if you give me a good reason to do like that :D
- It would be good not to dispatch an action if `cardPosition === selectedCardPosition` (I dont think a card should have the ability to be selected and targeted at the same time)